generated/nidigital/nidigital/_library.py

```diff
@@ -6,6 +6,8 @@
 
 from nidigital._visatype import *  # noqa: F403,H303
 
+import nidigital.history_ram_cycle_information as history_ram_cycle_information  # noqa: F401
+
```

Comment:
why is this needed by `_library.py`?

Reply:
In this case it's not needed. But in the case of other custom types (like `waveform_info` in niscope) the module also contains a type that's passed to the C function. Unconditionally adding the import keeps the codegen and/or metadata simpler.